### PR TITLE
Consume api

### PR DESCRIPTION
**Pull request by olawale-o**
## The following was implemented:
- Create a new game with the name of my choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
+    };
+    await fetchScore(postData);
+    user.value = '';
+    score.value = '';
+    const { result } = await refreshScore(gameId);
+    updateDomWithScores(scores, result);
+    user.focus();
+  });
+
+  refreshButton.addEventListener('click', async (event) => {
```
Comment (bellom) [inline src/index.js:44]: Let's double-check on this function, we want the score list to change once the `refresh` button is clicked.

In your case, clicking the `refresh` button doesn't change the score.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @olawale-o  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (bellom) [reply src/index.js:44]: @olawale-o Apology for the mixed up 🙏 , there was a little glitch at my end, everything seems working fine.

I will be here to make sure this PR is approved.

Comment (olawale-o) [reply src/index.js:44]: Hi @bellom

I really appreciate your time and effort spent on the review. I believe I understand the reason for the mix up. It is totally fine. 

Please keep up the good work. Thanks

Comment (bellom) [reply src/index.js:44]: What a professional reply 👍 , please keep it up.

**Review by bellom — APPROVED** [review]
## Hi @olawale-o  👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**